TerminalApp/Controller/LoginViewController.swift

```diff
@@ -22,23 +29,73 @@ class LoginViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        // インジケーターの表示設定
+        loadingView.backgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.5)
+        indicator.center = view.center
+        indicator.style = .large
+        indicator.color = .white
+        indicator.hidesWhenStopped = true // アニメーション停止時に非表示
+        
+        loadingView.addSubview(indicator) // ロード画面にインジケーターを追加
+        
     }
 
     @objc func tapButton(_: UIResponder) {
-        if hostname.text == "" || username.text == "" || password.text == "" || port.text == "" {
-            let alert = UIAlertController(title: "入力エラー", message: "全ての項目は必須入力です", preferredStyle: .alert)
+        
+        guard let host = hostname.text, let user = username.text, let pass = password.text, let portNum = port.text else { return }
```

Comment:
guardにエラー処理をまとめられそうな気がします。

```suggestion
        guard let host = hostname.text, !host.isEmpty let user = username.text, !user.isEmpty, let pass = password.text, !pass.isEmpty, let portNum = port.text, portNum.isEmpty else { 
        let alert = Alert(title: "入力エラー", message: "全ての項目は必須入力です")
         present(alert.controller, animated: true, completion: nil)
        return 
}
```